### ISSUE TITLE
Chewing audio

### ISSUE THREAD

**Issue by Lanimel** [telegramdesktop/tdesktop#8819]
# Description
With the arrival of the last beta, I ran into an interesting error. Telegram began "to chew" on any audio track, be it music or video. It looks like a small lags about every 3 seconds. I got the latest updates and the stable version, but the error persists. This works for any media content with my earbuds and is repeated on two computers running windows 8 and 8.1, respectively. The earbuds are working fine and work great with other applications, and reinstalling the driver was unsuccessful. On-ear headphones don't have this problem

# Logs
Here are the only logs when switching between headphones:

```Audio Info: OnDeviceStateChanged("{0.0.0.00000000}.{//device_id}", 1) scheduling detach from audio device.
Audio Info: OnDefaultDeviceChanged() scheduling detach from audio device, flow 0, role 0, new_default_device_id: "{0.0.0.00000000}.{//device_id}"
Audio Info: skipping OnDefaultDeviceChanged() flow 0, role 1, new_default_device_id: "{0.0.0.00000000}.{//device_id}"
Audio Info: OnDefaultDeviceChanged() scheduling detach from audio device, flow 0, role 2, new_default_device_id: "{0.0.0.00000000}.{//device_id}"
Audio Info: recreating audio device and reattaching the tracks
Skipping message, because it is already in blocks!
Skipping message, because it is already in blocks!
```

# System info
Telegram 2.4.3
Windows 8.1
Earbuds: Panasonic RP-HJE118GUK

**Comment by LeonTheOriginal** [comment]
It seems that this (or a similar) issue occurs on Win 10 since Version 2.3.1 (but not 2.3.0).

**Comment by maestro9** [comment]
Same. 2.4.2 / Win 10

**Comment by Lanimel** [comment]
I got a fix for this problem with version 2.4.4
